<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColumnReferenceSelector</class>
 <widget class="QFrame" name="ColumnReferenceSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select column references</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="textSearch">
        <property name="placeholderText">
         <string>Search...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAdd">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="listView">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
